### DISCUSSION TITLE
Preferred media type for GEDCOM files?

### DISCUSSION THREAD

**Discussion by cecilia-donnelly** [FamilySearch/GEDCOM#112] [Q&A] [answered]
Hi there,
I'm working on adding GEDCOM support to permanent.org.  We set a media type / MIME type on files that we store.  What's the preferred type to use for gedcom and gedzip files?
Thanks!

**Comment by clarkegj** [comment]
Cecilia,

Check out:
https://www.iana.org/assignments/media-types/application/vnd.familysearch.gedcom+zip

See some extensions overview at FileInfo.com.
https://fileinfo.com/extension/ged
https://fileinfo.com/extension/gdz

Let me know if you need anything else.

Gordon
From: Cecilia Donnelly Krum ***@***.***>
Sent: Tuesday, February 15, 2022 2:49 PM
To: FamilySearch/GEDCOM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [FamilySearch/GEDCOM] Preferred media type for GEDCOM files? (Discussion #112)


Hi there,
I'm working on adding GEDCOM support to permanent.org. We set a media type / MIME type on files that we store. What's the preferred type to use for gedcom and gedzip files?
Thanks!

-
Reply to this email directly, view it on GitHub<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FFamilySearch%2FGEDCOM%2Fdiscussions%2F112&data=04%7C01%7Cgordon%40familysearch.org%7C09e38592e8c74c76715208d9f0cd052a%7C61e6eeb35fd74aaaae3c61e8deb09b79%7C0%7C0%7C637805585636583350%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=DR1QhMBJwGBLHA1YbB5cn5uPmaqIG8QZiYOhSTHyLJA%3D&reserved=0>, or unsubscribe<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAGMW7MHLA574KJ2JPBVJ3DU3LC55ANCNFSM5OPZQBDQ&data=04%7C01%7Cgordon%40familysearch.org%7C09e38592e8c74c76715208d9f0cd052a%7C61e6eeb35fd74aaaae3c61e8deb09b79%7C0%7C0%7C637805585636583350%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=4ny6col9nSMejFcZuS1umks74I7Knj8zbxEecKZCEdU%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cgordon%40familysearch.org%7C09e38592e8c74c76715208d9f0cd052a%7C61e6eeb35fd74aaaae3c61e8deb09b79%7C0%7C0%7C637805585636583350%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=1WXrjtD2oDe1A7nSEb9VI9rLlwyf3dX94yE%2BAVmPMNk%3D&reserved=0> or Android<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cgordon%40familysearch.org%7C09e38592e8c74c76715208d9f0cd052a%7C61e6eeb35fd74aaaae3c61e8deb09b79%7C0%7C0%7C637805585636583350%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Azin7HbISaQm%2BuUNjgkoW%2BrEetVqfaD5e7Zj%2BeMFkEc%3D&reserved=0>.

**Comment by dthaler** [comment, marked as answer]
> We set a media type / MIME type on files that we store. What's the preferred type to use for gedcom and gedzip files?

GEDCOM files: [text/vnd.familysearch.gedcom](https://www.iana.org/assignments/media-types/text/vnd.familysearch.gedcom)
GEDZIP files: [application/vnd.familysearch.gedcom+zip](https://www.iana.org/assignments/media-types/application/vnd.familysearch.gedcom+zip)

See https://github.com/FamilySearch/GEDCOM/blob/main/version-detection/version-detection.md#iana-media-type-registration for the specification of these media types.

**Comment by cecilia-donnelly** [comment]
Fantastic!  Thank you both for the quick response. :)

**Comment by clarkegj** [comment]
Thanks Gordon!
Robert

__________________________
Robert Bryan Friedman, Ph.D.Executive Director
Permanent Legacy Foundation••••••••••••••••••••••••••••••••••••••••!!: schedule
a meeting with mem:  +1.512.887.4483
in: linkedin.com/in/robertbryanfriedman/








On Tue, Feb 15, 2022 at 6:52 PM Gordon Clarke ***@***.***>  wrote:
Cecilia,



Check out:

https://www.iana.org/assignments/media-types/application/vnd.familysearch.gedcom+zip



See some extensions overview at FileInfo.com.

https://fileinfo.com/extension/ged

https://fileinfo.com/extension/gdz



Let me know if you need anything else.



Gordon

From:  Cecilia Donnelly Krum ***@***.***> 
Sent:  Tuesday, February 15, 2022 2:49 PM
To:  FamilySearch/GEDCOM ***@***.***>
Cc:  Subscribed ***@***.***>
Subject:  [FamilySearch/GEDCOM] Preferred media type for GEDCOM files?
(Discussion #112)



Hi there,
I'm working on adding GEDCOM support to permanent.org. We set a media type /
MIME type on files that we store. What's the preferred type to use for gedcom
and gedzip files?
Thanks!

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS  or Android. 
You are receiving this because you are subscribed to this thread.Message ID: <
***@***.***>
